app/editor/src/features/navbar/NavBar.tsx

```diff
@@ -0,0 +1,48 @@
+import { Row } from 'components/flex/row';
+import { NavBarGroup, NavBarItem } from 'components/navbar';
+import { useLocation } from 'react-router-dom';
+import { Claim } from 'tno-core';
+
+/**
+ * The navigation bar that is used throughout the TNO editor application. Add or remove navigation bar items here.
+ */
+export const NavBar: React.FC = () => {
+  const location = useLocation();
+
+  const showEditor = location.pathname.startsWith('/contents');
+  const showAdmin = location.pathname.startsWith('/admin');
+
+  return (
+    <NavBarGroup>
+      <Row>
+        <NavBarItem navigateTo="/contents" label="Editor" claim={Claim.editor} />
+        <NavBarItem navigateTo="/admin" label="Admin" claim={Claim.administrator} />
+      </Row>
+      {showEditor && (
```

Comment:
I don't love this implementation, but it's the only one I could think of on short notice.  It's the new sub-menu navbar.